packages/components/src/components/SignIn.vue

```diff
@@ -384,7 +388,6 @@ export default {
   .your-data {
     display: flex;
     flex-direction: row;
-    align-items: center;
```

Comment:
When the "your data is your data" messaging collapses into rows, the shield is no longer aligned with this change
![image](https://github.com/user-attachments/assets/6129c880-6ad7-4028-855a-dae51f3407fe)

Reply:
Looks like there's a `@media (max-width: 365px)` query responsible for this somewhere.